src/main/java/eu/dissco/backend/service/AnnotationService.java

```diff
@@ -108,16 +115,8 @@ private AnnotationResponse parseToAnnotationResponse(JsonNode response) {
     );
   }
 
-  @NotNull
-  private static AnnotationEvent mapAnnotationRequestToEvent(AnnotationRequest annotation,
-      String userId) {
-    return new AnnotationEvent(annotation.type(), annotation.motivation(), userId, Instant.now(),
-        annotation.target(), annotation.body());
-  }
-
   public JsonApiWrapper updateAnnotation(String id, AnnotationRequest annotation, String userId,
-      String path)
-      throws NoAnnotationFoundException, JsonProcessingException {
+      String path) throws NoAnnotationFoundException {
     var result = repository.getAnnotationForUser(id, userId);
```

Comment:
probably a new US but it seems to me if you're doing a database call already, you might as well pull the whole annotation and check if the annotations are equal

Reply:
hmm yes maybe. So the processing service will check that as well but we might be able to reduce those calls when we do a check here. Ok seems logical, feel free to create US for this and pick it up with the annotation changes.

Reply:
https://naturalis.atlassian.net/browse/DD-761